inference-engine/tests/functional/inference_engine/serialization/single_layer/convolution.cpp

```diff
@@ -14,9 +14,7 @@ TEST_P(ConvolutionLayerTest, Serialize) {
 }
 
 const std::vector<InferenceEngine::Precision> netPrecisions = {
-    InferenceEngine::Precision::FP32, InferenceEngine::Precision::FP16,
-    InferenceEngine::Precision::I16, InferenceEngine::Precision::I32,
-    InferenceEngine::Precision::I64};
+    InferenceEngine::Precision::FP32, InferenceEngine::Precision::FP16};
```

Comment:
According to [spec](https://github.com/openvinotoolkit/openvino/blob/master/docs/ops/convolution/Convolution_1.md), for which inputs are of floating point type, instances of this op with integer element type are removed.

Reply:
@ggalieroc , why do you removed support for integer types if it actually worked in CPU plugin? What if there is some model with integer convolutions and after the change the model will stop working?

@dmitry-gorokhov , please, comment.

Reply:
Discussed with @dmitry-gorokhov and he agree with this change.